lessons/Algebra2/exploring-quadratic-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,214 @@
+|===
+
+== Looking for Patterns @duration{20 minutes}
+
+=== Overview
+Students explore the @starter-file{alg2-fuel-efficiency}, and create scatter plots to search for relationships between columns. They share and discuss their findings with the class, discovering the limitations of linear models.
+
+=== Launch
+
+When you're driving, what speed will save you the most money on gas?
+
+[.lesson-instruction]
+- Open the @starter-file{alg2-fuel-efficiency}
+- From the File menu, select "Save a Copy", and click "Run".
+- Working in pairs or small groups, complete questions 1-10 of @printable-exercise{pages/exploring-mpg.adoc}.
+
+@teacher{Review student answers to confirm that students see a _non-linear_ relationship in the dataset with a relatively strong correlation. **Make sure students have agreed on a "most efficient speed"**, as the location of the vertex is important for the next activity!}
+
+=== Investigate
+
+@center{@image{images/fuel-efficiency-quadratic.png}}
+
+There's definitely a relationship here. Even though some cars are more efficient than others, fuel efficiency generally increases from 15 to around 45-55mph. But after 55mph, fuel efficiency _decreases._
+
+[.lesson-instruction]
+- Let's try to build a linear model for this data.
+- Working in pairs or small groups, complete the rest of of @printable-exercise{pages/exploring-mpg.adoc}.
+
+By now we've tried to fit quite a few linear models to this data, and we're not getting anywhere.
+
+[cols="^.^1a,^.^1a,^.^1a", frame="none", stripes="none"]
+|===
+| We can build linear models that fit the increasing half of the data...
+| We can build linear models that fit the decreasing half of the data...
+| We can build linear models that try to split the difference...
+
+| @image{images/fuel-efficiency-linear1.png}
+| @image{images/fuel-efficiency-linear2.png}
+| @image{images/fuel-efficiency-linear3.png}
+|===
+
+...but all of them are so far away from so many points, and our @math{R^2} is pretty bad.
+
+=== Synthesize
+
+- Why didn't linear models work for us?
+- What kinds of curves did you draw to fit better?
+- Can you think of functions that look like those curves?
+
+== Quadratic Relationships @duration{10 minutes}
+
+=== Overview
+Students are introduced to quadratic relationships in the real world, then learn about them in abstract, mathematical terms. They develop vocabulary for describing parts of a parabola.
+
+=== Launch
+
+Linear models are great for fitting simple relationships. But the real world isn't so simple, and, more often than not, linear models are just too primitive to handle that complexity.
+
+Relationships shaped like this are called _quadratic relationships_. They form shapes that look like arches, called parabolas. Sometimes the data only includes the _rising_ part of the parabola, sometimes it only includes the _falling_ part, and and sometimes it includes both.
```

Comment:
It doesn't make sense to say `Relationships shaped like this are called` when you haven't talked about the shape yet.

Reply:
See edit.